gui/wxpython/mapdisp/properties.py

```diff
@@ -279,14 +279,17 @@ def _onToggleCheckBox(self, event):
             self.widget.SetLabel(self.defaultLabel)
 
 
-class RBShowInStatusbar:
+class RBShowInStatusbar(PropertyItem):
     """Radiobox managing widgets in statusbar."""
 
-    def __init__(self, parent, sbmanager):
+    def __init__(self, parent, mapWindowProperties, sbmanager):
+        PropertyItem.__init__(self, mapWindowProperties)
         self.name = "showInStatusbar"
         self.statusbarManager = sbmanager
 
-        choices = self.statusbarManager.GetItemLabels()
+        choices = self.statusbarManager.GetItemLabels(
+            self.statusbarManager.statusbarItems
+        )
```

Comment:
Passing the attribute like that is strange, maybe you want to add GetDisabledItemLabels method instead of passing the parameter here.